lib/widgets/layouts/dashboard_layout.dart

```diff
@@ -173,7 +174,7 @@ class DashboardLayoutState extends State<DashboardLayout>
       PaddedButton(
         color: getButtonColorByRoute(ReceiveTransactionScreen.route),
         padding: EdgeInsets.zero,
-        text: 'Receive',
+        text: AppLocalizations.of(context)!.receive,
```

Comment:
Add localization to 'Address copied', 'Copy address to clipboard', 'Settings', 'Current address' copies.